src/AutoRest.CSharp/Common/Output/Models/Types/ObjectType.cs

```diff
@@ -51,6 +51,27 @@ public IEnumerable<ObjectType> EnumerateHierarchy()
             }
         }
 
+        /// <summary>
+        /// Enumerates the inheritace hireachy.
+        /// </summary>
+        /// <returns>IEnumerable of inherited CSharpClass. </returns>
+        public IEnumerable<CSharpType> EnumerateInherits()
```

Comment:
can we reconcile the difference between this and EnumerateHierarchy?

Reply:
Indeed. I carefully compared them and found they are basically doing the same thing... so I removed it